doc/lib_term.md

```diff
@@ -23,7 +23,7 @@ Here is an example clock display from the examples folder, using
 the char_circle and char_line primitives to draw the clock face
 and the hands:
 
-![example image showing a clock](example_lib_term.png)
+![example image showing a clock](lib_term_illustration.png)
```

Comment:
If I understand it right this image file isn't something separate. It's part of lib_term documentation file. The new name puts them closer to one another in the file list.